components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/validators/AbstractResponseTypeRequestValidator.java

```diff
@@ -187,6 +193,46 @@ public OAuth2ClientValidationResponseDTO validateClientInfo(HttpServletRequest r
         }
     }
 
+    private void isHybridFlowRequest(HttpServletRequest request, OAuthAppDO appDO)
```

Comment:
Since this is a void method can we change the method as below?
```suggestion
    private void validateHybridFlowRequest(HttpServletRequest request, OAuthAppDO appDO)
```

Reply:
Fixed with: https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2461/commits/e89b7c23081c1c843e284226ecd37e1242f91ac0